core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala

```diff
@@ -125,9 +125,9 @@ class DataCutter(uid: String = UID[DataCutter]) extends Splitter(uid = uid) with
     val dataPrep = data
       .filter(r => labelSet.contains(r.getDouble(labelColIdx)))
       .withColumn(labelColName, data(labelColName).as(labelColName, metadataNA.toMetadata))
-
+    log.info(s"number of rows after only using top 100 labels ${dataPrep.count()}")
```

Comment:
Please use upper case in the first word, e.g. `Number of rows...` (same applies for other log statements.)